tasks/lammps/run.py

```diff
@@ -146,11 +153,20 @@ def faasm(ctx, repeats=1, nprocs=None, procrange=None):
 
 
 @task
-def native(ctx, repeats=1, nprocs=None, procrange=None):
+def native(ctx, bench, repeats=1, nprocs=None, procrange=None):
     """
     Run LAMMPS experiment on OpenMPI
     """
-    result_file = _init_csv_file("lammps_native.csv")
+    if bench not in BENCHMARKS:
+        print("Unrecognized benchmark: {}".format(bench))
+        print("Benchmark must be one in: {}".format(BENCHMARKS.keys()))
+        exit(1)
+
+    _bench = BENCHMARKS[bench]
```

Comment:
It feels like this bit of code is repeated in a few places, could we just have a funciton called `get_lammps_benchmark` or something?

Reply:
I have added a `get_lammps_benchmark(bench)` method in `tasks/lammps/env.py`